README.md

```diff
@@ -15,6 +15,32 @@ To install this module in Julia, use
 ] add https://github.com/oscar-system/GAP.jl
 ```
 
+## Basic usage
+
+After entering the following in Julia,
+```julia
+using GAP
+```
+one may call any GAP function by prefixing its name with `GAP.Globals.`. For example:
+```julia
+julia> GAP.Globals.SymmetricGroup( 3 )
```

Comment:
I am not going to request a change for this, but I note that once more, this is formatted differently in terms of whitespace than the other example inputs :-).

Reply:
I think because you made this line longer because of your suggestions. Anyway, fixed it.

Reply:
Still fixed it.